package.json

```diff
@@ -37,7 +37,7 @@
     "presystem-test": "npm run compile",
     "check": "gts check",
     "clean": "gts clean",
-    "compile": "tsc -p .",
+    "compile": "tsc -p . && cp src/types.d.ts build/src/",
```

Comment:
To keep things straight in the books - what do you think of installing `googleapis` as a dev dependency, then having this step copy the `d.ts` out of there?  Similar to what we do with `pbts`?

Reply:
Oh, I would be cool with that - I didn't realize that there were any protos/declarations available in `googleapis`.

Reply:
So I didn't find anything like what you were mentioning for BigQuery in googleapis. I'd also have some concerns about generating types via `pbjs` and `pbts` because I think it means we'd need to include protobufjs/long as a dependency, which doesn't make sense for BigQuery yet.

Reply:
Ohh, I thought you meant the googleapis *repo*, my bad.

I'm kind of less into this, it won't be as simple as a `cp` since that file isn't just types, but also a class that requires an additional dep to pull in external types and functions. Additionally some of the types have extra fields that are specific to `googleapis` (namely `auth`) which don't really apply here. 

The types I have here are just a copy/paste of those minus the `googleapis` specific fields and renamed to resemble the types in our grpc clients.

Reply:
I've always been advised against copying files out of `node_modules`, just because there's no guarantee it will be structured as you expect (as an example, perhaps there's two versions of a library, so the library you want is actually nested in a sub-folder -- in this case it doesn't see like that could happen, just always something I'm paranoid about.

If you do go this route, perhaps make a little build script that uses:

```
require.resolve('googleapis/build/srce/apis/bigquery/v2.d.ts')
```